### PR TITLE
PIN-3678 Add IPA code to Risk Analysis PDF

### PR DESCRIPTION
**Pull request by nttdata-rtorsoli**

### CODE REVIEW

File: src/main/scala/it/pagopa/interop/purposeprocess/api/impl/PurposeVersionActivation.scala [hunk 1]
```diff
@@ -212,13 +212,15 @@ final case class PurposeVersionActivation(
   )(implicit contexts: Seq[(String, String)]): Future[PurposeVersion] = {
     val documentId: UUID = uuidSupplier.get()
     for {
-      (producerDescription, consumerDescription) <- getTenantName(eService.producerId).zip(
-        getTenantName(purpose.consumerId)
-      )
+      ((producerDescription, producerValue), (consumerDescription, consumerValue)) <- getTenantTuple(
+        eService.producerId
+      ).zip(getTenantTuple(purpose.consumerId))
```
Comment (galales) [inline src/main/scala/it/pagopa/interop/purposeprocess/api/impl/PurposeVersionActivation.scala]: IMO this became less readable, I would suggest something like
```suggestion
      (producer, consumer) <- getTenant(eService.producerId).zip(getTenant(purpose.consumerId))
```

File: src/main/resources/riskAnalysisTemplate/index.html [hunk 2]
```diff
@@ -51,12 +51,12 @@ <h1>Analisi del rischio</h1>
 
           <div class="item">
             <div class="label">Erogatore</div>
-            <div class="value">${producerName}</div>
+            <div class="value">${producerName} (codice IPA: ${producerValue})</div>
```
Comment (galales) [inline src/main/resources/riskAnalysisTemplate/index.html]:
```suggestion
            <div class="value">${producerName} (codice IPA: ${producerIPACode})</div>
```

File: src/main/resources/riskAnalysisTemplate/index.html [hunk 3]
```diff
@@ -51,12 +51,12 @@ <h1>Analisi del rischio</h1>
...
           </div>
 
           <div class="item">
             <div class="label">Fruitore</div>
-            <div class="value">${consumerName}</div>
+            <div class="value">${consumerName} (codice IPA: ${consumerValue})</div>
```
Comment (galales) [inline src/main/resources/riskAnalysisTemplate/index.html]:
```suggestion
            <div class="value">${consumerName} (codice IPA: ${consumerIPACode})</div>
```

File: src/main/scala/it/pagopa/interop/purposeprocess/model/riskAnalysisTemplate/EServiceInfo.scala [hunk 4]
```diff
@@ -1,3 +1,9 @@
 package it.pagopa.interop.purposeprocess.model.riskAnalysisTemplate
 
-final case class EServiceInfo(name: String, producerName: String, consumerName: String)
+final case class EServiceInfo(
+  name: String,
+  producerName: String,
+  producerValue: String,
```
Comment (galales) [inline src/main/scala/it/pagopa/interop/purposeprocess/model/riskAnalysisTemplate/EServiceInfo.scala]:
```suggestion
  producerIPACode: String,
```

File: src/main/scala/it/pagopa/interop/purposeprocess/model/riskAnalysisTemplate/EServiceInfo.scala [hunk 5]
```diff
@@ -1,3 +1,9 @@
...
+  consumerName: String,
+  consumerValue: String
```
Comment (galales) [inline src/main/scala/it/pagopa/interop/purposeprocess/model/riskAnalysisTemplate/EServiceInfo.scala]:
```suggestion
  consumerIPACode: String
```

**Review by galales — COMMENTED** [review]
This works perfectly for public administrations, but what happens with private organizations that are not in IPA? (the PDF says `codice IPA`).

cc @beetlecrunch

**Comment by beetlecrunch** [comment]
> This works perfectly for public administrations, but what happens with private organizations that are not in IPA? (the PDF says `codice IPA`).
> 
> cc @beetlecrunch

Maybe we can apply this behavior:
- if tenant.extarnalId.origin == "IPA" then (codice IPA: ${producerValue})
- else ""

**Review by beetlecrunch — APPROVED** [review]

**Review by galales — APPROVED** [review]

**Pull request merged**